.changeset/tricky-poems-yell.md

```diff
@@ -0,0 +1,7 @@
+---
+"@clerk/clerk-js": patch
+"@clerk/elements": patch
+"@clerk/shared": patch
+---
+
+Move `iconImageUrl` util to `shared` package.
```

Comment:
We could also mention that this is an internal change with no user-facing impact. It helps especially when the patch involves different packages on the same changeset. 

Not a strong opinion so feel free to use your judgment here.

Reply:
👍🏼 updated